### ISSUE TITLE
Fix TODO in `docs/index.md`

### ISSUE THREAD

**Issue by ahouseholder** [CERTCC/SSVC#494]
Remove this blurb before release.

> {== TODO remove this warning when the site is ready ==}
> !!! warning "Work in Progress"
> 
>     This website is a work in progress.
>     While it is intended to eventually be the official documentation for SSVC, it is not yet complete.
>     For the latest version of SSVC documentation, see the [GitHub repository](https://github.com/CERTCC/SSVC).
> 

**Note** this issue needs to be a separate PR from anything else so that we can be selective about when we merge it (I.e., we can have it tee'd up in advance, but we shouldn't merge until we're nearly done.)

**Comment by ahouseholder** [comment]
https://github.com/CERTCC/SSVC/blob/5915548b5f7324a1f0c3ce7dd2efec062a5883f5/docs/index.md?plain=1#L9-L14
